.github/workflows/tests.yml

```diff
@@ -16,21 +16,21 @@ on:
     branches: master
   schedule:
     # * is a special character in YAML so you have to quote this string
-    - cron:  '0 3 * * 6'
+    - cron: "0 3 * * 6"
   workflow_dispatch:
     inputs:
       reason:
-        description: 'Reason'
+        description: "Reason"
         required: false
-        default: 'Manual trigger'
+        default: "Manual trigger"
 
 jobs:
   Tests:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-          python-version: [3.7, 3.8, 3.9]
-          requirements-level: [pypi]
+        python-version: [3.7, 3.8, 3.9, 3.10, 3.11, 3.12]
```

Comment:
you have to put "3.12" under quotes otherwise it doesn't work

Reply:
and @ntarocco said the tested versions should be the min "3.9" and the max "3.12" to shrink down the combination list

Reply:
> you have to put "3.12" under quotes otherwise it doesn't work

it seems working, no?

Reply:
> you have to put "3.12" under quotes otherwise it doesn't work

it applies only to `3.10`, because it evaluates to `3.1` in GH Actions ☺️

Reply:
oh, that is strange. it thought it has to applied to all of the "decimals", because also 3.12 would evaluate to 3.1.

Reply:
good to know so i don't have to quote in the other PR's